sql_metadb/derived_tables/finance_invoice_transactions.sql

```diff
@@ -0,0 +1,133 @@
+CREATE TABLE finance_invoice_transactions AS
+WITH invoice AS (
+    SELECT
+        invoices.id AS invoice_id,
+        jsonb_extract_path_text(invoices.jsonb, 'folioInvoiceNo') AS folio_invoice_number,
+        jsonb_extract_path_text(invoices.jsonb, 'vendorInvoiceNo') AS vendor_invoice_number,
+        jsonb_extract_path_text(invoices.jsonb, 'invoiceDate')::timestamptz AS invoice_date,
+        jsonb_extract_path_text(invoices.jsonb, 'currency') AS invoice_currency,
+        jsonb_extract_path_text(invoices.jsonb, 'exchangeRate')::numeric(19, 14) AS invoice_exchange_rate,
+        jsonb_extract_path_text(invoices.jsonb, 'status') AS invoice_status,
+        invoice_lines.id AS invoice_line_id,
+        jsonb_extract_path_text(invoice_lines.jsonb, 'total')::numeric(19, 4) AS invoice_line_total,
+        jsonb_extract_path_text(jsonb_array_elements(invoice_lines.jsonb -> 'fundDistributions'), 'fundId')::uuid AS invoice_line_fund_id,
+        jsonb_extract_path_text(jsonb_array_elements(invoice_lines.jsonb -> 'fundDistributions'), 'value')::numeric(19, 4) AS invoice_line_distribution_value,
+        jsonb_extract_path_text(jsonb_array_elements(invoice_lines.jsonb -> 'fundDistributions'), 'distributionType') AS invoice_line_distribution_type,
+        organizations.id AS vendor_id,
+        jsonb_extract_path_text(organizations.jsonb, 'name') AS vendor_name
+    FROM
+        folio_invoice.invoice_lines AS invoice_lines
+        LEFT JOIN folio_invoice.invoices AS invoices ON invoices.id = invoice_lines.invoiceid
+        LEFT JOIN folio_organizations.organizations AS organizations ON jsonb_extract_path_text(invoices.jsonb, 'vendorId')::uuid = organizations.id
+),
+finance AS (
+    SELECT
+        fiscal_year.id AS fiscal_year_id,
+        jsonb_extract_path_text(fiscal_year.jsonb, 'code') AS fiscal_year,
+        ledger.id AS ledger_id,
+        jsonb_extract_path_text(ledger.jsonb, 'name') AS ledger_name,
+        budget.id AS budget_id,
+        jsonb_extract_path_text(budget.jsonb, 'name') AS budget_name,
+        fund.id AS fund_id,
+        jsonb_extract_path_text(fund.jsonb, 'code') AS fund_code
+    FROM
+        folio_finance.fiscal_year AS fiscal_year
+        LEFT JOIN folio_finance.ledger AS ledger ON ledger.fiscalyearoneid = fiscal_year.id
+        LEFT JOIN folio_finance.fund AS fund ON fund.ledgerid = ledger.id
+        LEFT JOIN folio_finance.budget AS budget ON budget.fundid = fund.id
+            AND budget.fiscalyearid = fiscal_year.id
+),
+transactions AS (
+    SELECT
+        transaction.id AS transaction_id,
+        jsonb_extract_path_text(transaction.jsonb, 'fromFundId')::uuid AS fromFundId,
+        jsonb_extract_path_text(transaction.jsonb, 'amount')::numeric(19, 4) AS transaction_amount,
+        jsonb_extract_path_text(transaction.jsonb, 'currency') AS transaction_currency,
+        jsonb_extract_path_text(transaction.jsonb, 'sourceInvoiceLineId')::uuid AS sourceInvoiceLineId
```

Comment:
For various reasons we use all lowercase identifiers in Postgres by convention.  So fromFundId would typically be written as from_fund_id, and sourceInvoiceLineId as source_invoice_line_id.  I will make these changes before merging.